frontend/public/components/utils/firehose.jsx

```diff
@@ -19,7 +19,9 @@ const shallowMapEquals = (a, b) => {
   return a.every((v, k) => b.get(k) === v);
 };
 
-const processReduxId = ({ k8s }, props) => {
+const CACHE_SYMBOL = Symbol('_cachedToJSResult');
```

Comment:
Safe to use the same symbol as the watcher. However since `firehose` is old and would probably be more grief to share the symbol with the sdk.

Reply:
I added this to the SDK because we have already an import from firehose.jsx to the SDK. I'm not sure what the "right' import direction here....?!